src/modal/docs/readme.md

```diff
@@ -1,4 +1,4 @@
-`$modal` is a service to quickly create AngularJS-powered modal windows.
+The `$modal` is an AngularJS `service` (a `provider`) to quickly create AngularJS-powered modal windows.
```

Comment:
A service isn't the same as a provider, so I think the parenthesis can be left out. Also "service" shouldn't need to be code (between `).